### ISSUE TITLE
IE8 Fails Due to Call to getComputedStyle

### ISSUE THREAD

**Issue by uncultured** [silviomoreto/bootstrap-select#1073]
Line 742:
```
menuStyle = getComputedStyle ? getComputedStyle(menu) : false,
```
Results in the following error in IE8:
```
'getComputedStyle' is undefined
```

Using a globally scoped polyfill messes with jQuery (it runs non-IE8-safe code if getComputedStyle is defined), so I got around this by doing the following:
```
 var getComputedStyle = window.getComputedStyle || function(el, pseudo) {
        this.el = el;
        this.getPropertyValue = function(prop) {
          var re = /(\-([a-z]){1})/g;
          if (prop == 'float') prop = 'styleFloat';
          if (re.test(prop)) {
            prop = prop.replace(re, function () {
              return arguments[2].toUpperCase();
            });
          }
          return el.currentStyle[prop] ? el.currentStyle[prop] : null;
        };
        return this;
      };
      var liHeight = a.offsetHeight,
          headerHeight = header ? header.offsetHeight : 0,
          searchHeight = search ? search.offsetHeight : 0,
          actionsHeight = actions ? actions.offsetHeight : 0,
          doneButtonHeight = doneButton ? doneButton.offsetHeight : 0,
          dividerHeight = $(divider).outerHeight(true),
          // fall back to jQuery if getComputedStyle is not supported
          menuStyle = getComputedStyle(menu),
          $menu = menuStyle ? $(menu) : null,
          menuPadding = parseInt(menuStyle ? menuStyle.paddingTop : $menu.css('paddingTop')) +
                        parseInt(menuStyle ? menuStyle.paddingBottom : $menu.css('paddingBottom')) +
                        parseInt(menuStyle ? menuStyle.borderTopWidth : $menu.css('borderTopWidth')) +
                        parseInt(menuStyle ? menuStyle.borderBottomWidth : $menu.css('borderBottomWidth')),
          menuExtras =  menuPadding + 
                        parseInt(menuStyle ? menuStyle.marginTop : $menu.css('marginTop')) + 
                        parseInt(menuStyle ? menuStyle.marginBottom : $menu.css('marginBottom')) + 2;
```

This creates a locally scoped function called getComputedStyle if the globally-scoped version on the window object isn't present. We want it locally scoped only because, like I mentioned before, jQuery does bad things if it finds a globally scoped version.

**Comment by caseyjhol** [comment]
Please try the latest version in master (1.7.3). This was resolved in 7b1bc3156155461e58c93fda9cd35842e02c85e1.

**Comment by uncultured** [comment]
The issue with that fix is that menuStyle then is set to false, which means $menu is set to null, which means lines 725-731 die when calling the css() method on $menu.

**Comment by caseyjhol** [comment]
No, that was fixed in the previous commit 26fe92cd1dd21ac83277bcd5f96c54d560e4b08c.

**Comment by uncultured** [comment]
Just checked out 1.7.3 and all this is fixed. I really appreciate the quick response but I'm sorry to have wasted your time!
